src/helpers/LinkButtonComponentType.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * The proptypes for {@link LinkButtonComponentType}.
+ *
+ * There is an additional required prop missing from this interface,
+ * `onKeyDown`, which follows the usual React interface
+ * `React.KeyboardEventHandler`.
+ */
+export interface LinkButtonComponentTypeProps {
+  id?: string;
+  className?: string;
+  /**
+   * Use for the main call to action on your service's start page. Adjusts
+   * style of button and adds a start icon.
+   */
```

Comment:
Remember this interface is for the benefit of users wrapping their own link components, so it needs to explain how this flag alters the properties, not why you would use it.